packages/reports/tests/integration/components/navi-date-picker-test.js

```diff
@@ -89,25 +79,105 @@ module('Integration | Component | Navi Date Picker', function(hooks) {
+    await render(hbs`<NaviDatePicker @date={{this.date}} @dateTimePeriod={{this.dateTimePeriod}} />`);
 
     assert.equal(getMonth(), 'July', 'The center date starts as the passed in date');
-
-    await click('.ember-power-calendar-nav-control--previous');
+    await clickPrevious();
     assert.equal(getMonth(), 'June', 'The center date changes to previous month');
-
-    await click('.ember-power-calendar-nav-control--next');
+    await clickNext();
     assert.equal(getMonth(), 'July', 'The center date goes back to original after click');
+    await clickNext();
+    assert.equal(getMonth(), 'August', 'The center date goes one month ahead');
+  });
+
+  test('Change center date - week', async function(assert) {
+    assert.expect(4);
+
+    this.set('date', moment('2015-07-14', TEST_FORMAT));
+    this.set('dateTimePeriod', 'week');
 
-    await click('.ember-power-calendar-nav-control--next');
+    await render(hbs`<NaviDatePicker @date={{this.date}} @dateTimePeriod={{this.dateTimePeriod}} />`);
+
+    assert.equal(getMonth(), 'July', 'The center date starts as the passed in date');
+    await clickPrevious();
+    assert.equal(getMonth(), 'June', 'The center date changes to previous month');
+    await clickNext();
+    assert.equal(getMonth(), 'July', 'The center date goes back to original after click');
+    await clickNext();
     assert.equal(getMonth(), 'August', 'The center date goes one month ahead');
   });
 
+  test('Change center date - month', async function(assert) {
+    assert.expect(4);
+
+    this.set('date', moment('2015-07-14', TEST_FORMAT));
+    this.set('dateTimePeriod', 'month');
+
+    await render(hbs`<NaviDatePicker @date={{this.date}} @dateTimePeriod={{this.dateTimePeriod}} />`);
+
+    assert
+      .dom('.ember-power-calendar-selector-nav-title')
+      .hasText('2015', 'The center date starts as the passed in date');
+    await clickPrevious();
+    assert.dom('.ember-power-calendar-selector-nav-title').hasText('2014', 'The center date changes to previous month');
+    await clickNext();
+    assert
+      .dom('.ember-power-calendar-selector-nav-title')
+      .hasText('2015', 'The center date goes back to original after click');
+    await clickNext();
+    assert.dom('.ember-power-calendar-selector-nav-title').hasText('2016', 'The center date goes one month ahead');
+  });
+
+  test('Change center date - quarter', async function(assert) {
+    assert.expect(4);
+
+    this.set('date', moment('2015-07-14', TEST_FORMAT));
+    this.set('dateTimePeriod', 'quarter');
+
+    await render(hbs`<NaviDatePicker @date={{this.date}} @dateTimePeriod={{this.dateTimePeriod}} />`);
+
+    assert
+      .dom('.ember-power-calendar-selector-nav-title')
+      .hasText('2015', 'The center date starts as the passed in date');
+    await clickPrevious();
+    assert.dom('.ember-power-calendar-selector-nav-title').hasText('2014', 'The center date changes to previous month');
```

Comment:
should these assert messages be specifying `quarter` or `year` instead of `month`?